### ISSUE TITLE
Update help text for noEmitOnError

### ISSUE THREAD

**Issue by danquirk** [Microsoft/TypeScript#3134]
tsc -help reports
```
--noEmitOnError         Do not emit outputs if any type checking errors were reported.
```
We no longer distinguish between type checking and other errors here so the help should simply say 'any errors were reported.' We'll need to update the wiki appropriately as well.
